fw/http.c

```diff
@@ -1558,6 +1559,7 @@ __tfw_http_resp_fwd_stale_noresp(TfwHttpReq *req)
 
 	return true;
 }
+ALLOW_ERROR_INJECTION(__tfw_http_resp_fwd_stale_noresp, NULL);
```

Comment:
I am not sure that it is ok to return NULL for error injection if this function really returns bool value, may be it is better to make it return int value and return -ENOMEM or any other error on fail if we want to use error injection tests.